lib/international-interventions-parser.ts

```diff
@@ -85,3 +85,38 @@ function getIsoCode(alpha3: string): string {
   }
   return ''
 }
+
+/**
+ * Validates that the new table is not smaller than the old table.
+ * The assumption is that our fetch data size will never shrink.
+ *
+ * @db the database to run the queries on
+ * @param origTable The original table to check
+ * @param newTable The new table to replace the original table
+ * @param ignore If true, ignore this check
+ *
+ * @throws Error if the new table is smaller than the old table.
+ */
+export async function validateTableLength(
```

Comment:
**Concern:** I am good with the behavior of this function, but I don't feel like it really belongs in this file? It doesn't really have anything to do with `international-interventions` or parsing, which for me at least caused confusion when it is later called in the actual fetch script.

Reply:
That's fine. I'll move it elsewhere. I'm mostly trying to keep the `fetch-recorded-data` script simple.